### ISSUE TITLE
Chicken after SC

### ISSUE THREAD

**Issue by Vivivil** [WarEmu/WarBugs#11040]
Expected behavior and actual behavior:
Queued for SC in Altdorf at level 15 90+%, leveled up via Gotrek quest then SC popped. During SC my character was playable but debolstered to 15. After the SC I became a chicken in Altdorf, dying did not fix.

Steps to reproduce the problem:
Queue for SC at high level % before next tier, level up on quest before SC pops. Play SC then after get stuck as a chicken?

Screenshots/Videos or archive.org evidences:

![vivivia_000](https://user-images.githubusercontent.com/34780913/34314176-02f14c44-e769-11e7-9bec-a9bda22a02f2.jpg)

* Complete rules for the bugtracker https://www.returnofreckoning.com/forum/viewtopic.php?f=52&t=10366

**Comment by CircuitByrd** [comment]
Didn't go so far as to take a video, but I've found this seems to also be reproducible for me if I click the "Leave" button too quickly. If I wait around for a minute before doing so, I do not. I can capture a video if that will be useful.

My belief, based on this, is that it occurs because the game thinks you "abandoned" the scenario. Delaying the appearance of the leave timer, changing how it checks this, 

Regardless, once chickened, death does not clear it, only logging out and back in.

![2019-12-03 15_38_46-Warhammer_ Age of Reckoning, Version 1 4 8, Copyright 2001-2012 Electronic Arts,](https://user-images.githubusercontent.com/26975895/70092237-713ffd80-15e3-11ea-9b87-34ed9f92ef1e.png)

**Comment by raptrr** [comment]
Would you be so kind to record it? I'll also try to reproduce it but a video would help us a lot.

**Comment by CircuitByrd** [comment]
Well, I had this happen in 3/5 scenarios during EU prime time yesterday playing on my Archmage, but then recorded four RvR matches in NA late night on my Swordmaster and couldn't get it to happen a single time, no matter how quickly I clicked the "leave" button, so there must be variables I'm not accounting for.
My Archmage has leveled out of T1 now, and I was in a different zone when queuing/joining. I don't recall if it happened after I leveled into T2, but it definitely happened in several scenarios in T1 where I didn't level, so that wasn't the variable. 
I'll keep trying to provide a video, but also to isolate exactly under what conditions this is happening.
